docs/docs/additional_resources/tutorials.mdx

```diff
@@ -1,15 +1,19 @@
 # Tutorials
 
-Below are links to tutorials and courses on LangChain. For written guides on common use cases for LangChain, check out the [use cases guides](/docs/use_cases).
+Below are links to tutorials and courses on [LangChain](https://en.wikipedia.org/wiki/LangChain). 
```

Comment:
feel like we probably don't need to link to wikipedia from our own docs? if wikipedia is better at describing us than our docs than 😅

Reply:
As you wish. I thought the new Wikipedia link is great.